### PR TITLE
Add vehicles

### PR DESCRIPTION
**Pull request by kalebhermes**
## Connection with issue(s)
No open issue to add Vehicles. I needed it for a personal project and figured it was worth sharing. 

## Testing and Review Notes
Running unit tests should cover all cases

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ environment:
 dependencies:
   crypto: ^3.0.0
 dev_dependencies:
-  test: ^1.16.0-nullsafety.17
+  test: ^1.16.0-nullsafety.17
```
Comment (drager) [inline pubspec.yaml]: Should keep this newline

**Review by drager — APPROVED** [review]
Thanks a lot! Looks good, run `dartfmt` and then we can merge

File: pubspec.yaml (hunk 1 above)
Comment (kalebhermes) [reply pubspec.yaml]: Done!

**Comment by kalebhermes** [comment]
`dartfmt` run, newline added to the end of `pubspec.yml`

**Comment by drager** [comment]
@kalebhermes Sorry, merged another PR before yours. Could you merge/rebase master into your branch? :pray:

**Comment by kalebhermes** [comment]
@drager It's done!

**Comment by kalebhermes** [comment]
@drager bump

**Comment by kalebhermes** [comment]
Bump again ;)

**Comment by kalebhermes** [comment]
merge

**Comment by drager** [comment]
Haha @kalebhermes! :smile: Sorry for being slow. Let's merge it! :tada:

**Comment by kalebhermes** [comment]
🎉 🚢

**Pull request merged**